### DISCUSSION TITLE
idea : Packaging for windows (and maybe others ) through github actions

### DISCUSSION THREAD

**Discussion by selmen2004** [thonny/thonny#3170] [General]
i currently run a python distribution with containing thonny and other libs for offline usage  ( github.com/selmen2004/PyTN ) 
I succeeded these days to run the whole process as a github action so I thought maybe it would be a good idea to the same for thonny

**Comment by aivarannamaa** [comment]
Thanks for the suggestion, but at the moment the scripts I use for packaging are not too much hassle either. Packaging and uploading for Windows might benefit from GitHub actions, but for macOS I need to sign the parts and notarize the result and I prefer keeping my private keys and passwords in my computer.

**Reply by RivetCode123** [reply]
Aivar, Sorry if I missed something obvious but I can't find the windows install ANYWHERE.  Is there an EXE package or do I download this entire GIT and do what with it?  Or is there a link I missed?    I found "pip install thonny" and did that, ... and it did stuff...  wonderful stuff I'm sure... but definately no thonny.   I did get this: 

> "Successfully installed ast-serialize-0.4.0 astroid-4.0.4 asttokens-3.0.1 colorama-0.4.6 dill-0.4.1 docutils-0.22.4 isort-8.0.1 jedi-0.20.0 librt-0.11.0 mccabe-0.7.0 mypy-2.1.0 mypy_extensions-1.1.0 packaging-26.2 parso-0.8.7 pathspec-1.1.1 platformdirs-4.9.6 pylint-4.0.5 pyserial-3.5 send2trash-2.1.0 setuptools-82.0.1 thonny-5.0.0 tomlkit-0.15.0 typing_extensions-4.15.0 wheel-0.47.0"  


But where is thonny???   Why is there no simple install or instructions???  Certainly I'm not the only one dazed and confused by this. 
Signed - Frustrated In Fresno.

**Reply by dsssssssss9** [reply]
Try here.. 


https://github.com/thonny/thonny/releases
